OpenRA.Mods.Cnc/OpenRA.Mods.Cnc.csproj

```diff
@@ -158,6 +158,7 @@
     <Compile Include="Traits\Attack\AttackTDGunboatTurreted.cs" />
     <Compile Include="Traits\GpsDot.cs" />
     <Compile Include="Traits\ChronoshiftReturnTransformedActor.cs" />
+    <Compile Include="Traits\TransferTimedExternalConditionOnTransform.cs" />
```

Comment:
Why put this on CnC dll? Sounds generic enough to me. Both Transforms and ExternalCondition traits are on Common dll.

Reply:
Because this is a hacktrait and we don't intend to extend it to general. See IRC log.

Reply:
Same as above: The implementation has fundamental limitations (listed in the `[Desc]`) that would make this trait a nightmare to support generally, and which can't be fixed.   They are not a problem for our specific use case in RA, but i'm sure modders will soon run into them if we offered this for general use.